examples/connectivity/plot_power_connectome.py

```diff
@@ -0,0 +1,93 @@
+"""
+Extracting signals and plotting a connectome for the Power-264 seed-region atlas
+=============================================================
+
+This example shows how to extract signals from spherical seed-regions based
+on the Power-264 atlas (Power, 2011) and estimating a connectome using sparse
+inverse covariance.
+
+Power, Jonathan D., et al. "Functional network organization of the
+human brain." Neuron 72.4 (2011): 665-678.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from nilearn import datasets
+
+
+# fetch power atlas coords
+power = datasets.fetch_atlas_power_2011()
+power_coords = np.vstack((
+    power.rois['x'],
+    power.rois['y'],
+    power.rois['z'],
+)).T
+print(power_coords.shape)
+
+#fetch dataset
+adhd = datasets.fetch_adhd(n_subjects=1)
```

Comment:
We don't do the explicit plotting of the file paths etc. as we do it in other examples with fetchers.